File: src/main/csharp/HU.OOP.Week2.Opdracht31/Zwembad.cs
namespace HU.OOP.Week2.Opdracht31;

public class Zwembad
{
    public double Lengte { get; set; }
    public double Breedte { get; set; }
    public double Diepte { get; set; }

    public Zwembad(double breedte, double lengte, double diepte) : this()
    {
        this.Breedte = breedte;
        this.Lengte = lengte;
        this.Diepte = diepte;
    }

    public Zwembad()
    {
    }


    public double Inhoud => this.Diepte * this.Lengte * this.Breedte;


    public override String ToString()
    {
        return "Dit zwembad is " + this.Breedte + " meter breed, " + this.Lengte + " meter lang, en " + this.Diepte +
               " meter diep";
        //return String.format("Dit zwembad is %.1f meter breed, %.1f meter lang, en %.1f meter diep", this.breedte, this.lengte, this.diepte);
    }
}

File: src/main/java/nl/hu/oop/werkboek/week2/les1/opdracht1/Zwembad.java
package nl.hu.oop.werkboek.week2.les1.opdracht1;

public class Zwembad {
    private double lengte;
    private double breedte;
    private double diepte;

    public Zwembad(double breedte, double lengte, double diepte) {
        self();
        self.breedte = breedte;
        self.lengte = lengte;
        self.diepte = diepte;

    }

    public Zwembad() {

    }

    public double getBreedte() {
        return self.breedte;
    }

    public double getLengte() {
        return self.lengte;
    }

    public double getDiepte() {
        return self.diepte;
    }

    public double inhoud() {
        return self.diepte * self.lengte * self.breedte;
    }

    public void setBreedte(double b) {
        self.breedte = b;
    }

    public void setLengte(double l) {
        self.lengte = l;
    }

    public void setDiepte(double d) {
        self.diepte = d;
    }

    @Override
    public String toString() {
        return "Dit zwembad is " + self.breedte + " meter breed, " + self.lengte + " meter lang, en " + self.diepte + " meter diep";
        //return String.format("Dit zwembad is %.1f meter breed, %.1f meter lang, en %.1f meter diep", self.breedte, self.lengte, self.diepte);
    }
}
